router/appRest.js

```diff
@@ -503,12 +504,21 @@ function sendResponseWhatsapp(response,responseBody, recipient, textContent) {
 		},
 		json: true
 	};
-	request(options, function (error, response, body) {
-		// adding logs for debugging purposes only
-		LOG.info("Sending response to whatsapp is failing :", error);
-		LOG.info("Sending response to whatsapp is failing :", response);
-		LOG.info("Sending response to whatsapp is failing :", body);
-		if (error) throw new Error(error);
+	request(options, function (error, resp, body) {
+		const edata = {
```

Comment:
appRest.js should not have this logic right? 
As we discussed yesterday, SRP 
This method of logging telemetry should be separate telemetry logger method.